tools/local-network/local_network.py

```diff
@@ -31,7 +31,7 @@
 MOB_RELEASE = os.getenv('MOB_RELEASE', '1')
 CARGO_FLAGS = '--release'
 TARGET_DIR = 'target/release'
-WORK_DIR = '/tmp/mc-local-network'
+WORK_DIR = os.path.join(TARGET_DIR, '/tmp/mc-local-network')
```

Comment:
This should not have `/tmp/` there.

Reply:
I also believe it should be an absolute path so you probably need to do something like `os.path.abspath(os.path.join(PROJECT_DIR, TARGET_DIR, 'mc-local-network'))`.
Please ensure this actually runs and everything works.

Reply:
yes, thats where it should end up at. I am not sure a relative path would work unless everything is executed from inside PROJECT_DIR (which may or may not be the case)

Reply:
oh, maybe not!